pmoves/tools/pr_monitor.py

```
@@ -103,7 +103,42 @@ def _run_json(cmd: list[str]) -> Any:
 
 
+def _parse_origin_remote() -> str | None:
+    """Parse owner/repo from the origin remote URL.
+
+    Prefers origin over whatever `gh repo view` auto-detects, because checkouts
+    with both origin (fork) and upstream (parent) remotes can confuse gh's
+    auto-detection and return the upstream instead of the fork being worked on.
+    """
+    try:
+        proc = subprocess.run(
+            ["git", "config", "--get", "remote.origin.url"],
```

Ruff (0.15.9)

[error] Starting a process with a partial executable path

(S607)